lib/setupmanagers/hckclient.rb

```diff
@@ -116,7 +119,7 @@ def copy_dvl
 
     def machine_in_default_pool
```

Comment:
This method is no longer necessary; it can be simply replaced with `pool == 'Default Pool'`